includes/blocks/class-blocks.php

```diff
@@ -0,0 +1,12 @@
+<?php
+/**
+ * Newspack Block Theme Blocks.
+ *
+ * @package Newspack_Block_Theme
+ */
+
+namespace Newspack_Block_Theme;
+
+defined( 'ABSPATH' ) || exit;
+
+require_once NEWSPACK_BLOCK_THEME_FILE_PATH . '/includes/blocks/subtitle-block/class-subtitle-block.php';
```

Comment:
Is this file meant to ever become a class? I ask because of the file name.

Reply:
It's not – updated the filename in 9d27276d95e09474786d4d131bdc64bf5c223221